include/samurai/cell_list.hpp

```diff
@@ -17,7 +18,7 @@ namespace samurai
     // CellList definition //
     /////////////////////////
 
-    template<std::size_t dim_, class TInterval=Interval<int>, std::size_t max_size_ = 16>
+    template<std::size_t dim_, class TInterval=Interval<int>, std::size_t max_size_ = default_config::max_level>
```

Comment:
class TInterval = default_config::interval_t

Need to be changed also in level_cell_array.hpp

Reply:
good catch